dessia_common/datatools/cluster.py

```diff
@@ -97,10 +97,10 @@ def _write_str_prefix(self):
         prefix += (f"{len(self)} samples, {len(self.common_attributes)} features, {self.n_clusters} clusters")
         return prefix
 
-    def _get_printed_value(self, dessia_object: DessiaObject, attr: str):
+    def _get_printed_value(self, dessia_object_idx: int, attr: str):
```

Comment:
In order to avoid abbrevation in variable names, consider naming the argument `object_index` or even just `index`

Reply:
OK, done.
I kept index instead of i (that you advised) in the for loop